src/utils/sortAndFilterResults.ts

```diff
@@ -0,0 +1,102 @@
+import { SearchParametersType, UnprocessedResultsFromCRM } from '../types'
+
+type MatchTallies = {
+  [index: string]: number
+  neighbour: number
+  propertyType: number
+  propertyGroup: number
+}
+
+function sortForPropertyTypes (property: any, desiredPropertyTypes: any): {propertyTypeUpdateTally: number, propertyTypeMatch: boolean} {
```

Comment:
I don't think this name is accurate. No sorting is involved. Perhaps `matchesPropertyType`